aws/offsite/aurora-snapshots-fargate/aurora-verify-backups-stack.yml

```diff
@@ -136,7 +136,7 @@ Resources:
             Value: aurora-verify-backups-instance
           -
             Name: DB_SNAPSHOT_IDENTIFIER_PREFIX
-            Value: production
+            Value: autoscale-prod
```

Comment:
Maybe this should be a Stack Parameter or there should be a Tag on the Snapshot that identifies it as a production backup instead of relying on naming conventions.